cql3/restrictions/statement_restrictions.cc

```diff
@@ -1357,6 +1477,23 @@ void statement_restrictions::prepare_indexed(const schema& idx_tbl_schema, bool
     }
     // If we're here, it means the index cannot be on a partition column: process_partition_key_restrictions()
     // avoids indexing when _partition_range_is_simple.  See _idx_tbl_ck_prefix blurb for its composition.
```

Comment:
Don't separate this comment from the _idx_tbl_ck_prefix assignment and reservation below; the comment goes with that code.